build_libzmq.sh

```diff
@@ -17,7 +17,7 @@ export CFLAGS=-fPIC
 export CXXFLAGS=-fPIC
 export PKG_CONFIG_PATH=$ZMQ_PREFIX/lib/pkgconfig
 
-test -f zeromq-$ZMQ.tar.gz || wget https://github.com/$ZMQ_REPO/releases/download/v$ZMQ/zeromq-$ZMQ.tar.gz -O zeromq-$ZMQ.tar.gz
+test -f zeromq-$ZMQ.tar.gz || ZMQ=$ZMQ ZMQ_REPO=$ZMQ_REPO node "$(dirname $(realpath $0))/download-zmq.js"
```

Comment:
`../download-zmq.js` works for me. Because the script is executed in the `./zmq` folder.

But if I try to unpack it with `tar xzf zeromq-4.1.5.tar.gz`
I get this error:
```
tar: Unrecognized archive format
tar: Error exit delayed from previous errors.
```